wordcloud/wordcloud.py

```diff
@@ -404,8 +404,11 @@ def generate_from_frequencies(self, frequencies, max_font_size=None):
                                                max_font_size=self.height)
                 # find font sizes
                 sizes = [x[1] for x in self.layout_]
-                font_size = int(2 * sizes[0] * sizes[1] / (sizes[0]
-                                                           + sizes[1]))
+                try:
+                    font_size = int(2 * sizes[0] * sizes[1] 
```

Comment:
Could you please add a test for this, too? It didn't look like it was included, but I might have missed it.